pi-manage

```diff
@@ -1319,6 +1281,249 @@ def import_tokens(file, tokenrealm=None):
 token_manager.add_command('import', Command(import_tokens))
 
 
+def conf_export(conftype=DEFAULT_CONFTYPE_LIST, filename=None, name=None, stdout=True, print_passwords=False):
```

Comment:
And the default list is mutable which breaks some python constraints. You can use a tuple instead.

Reply:
thank you for the hint